shared_model/interfaces/common_objects/pager.hpp

```diff
@@ -77,6 +79,13 @@ namespace shared_model {
             .finalize();
       }
     };
+
+    /**
+     * Max limit
+     */
+    constexpr Pager::Limit MAX_PAGER_LIMIT = 100;
+    static_assert(MAX_PAGER_LIMIT == iroha::model::Pager::MAX_PAGER_LIMIT,
```

Comment:
What is the point to check this static assert? Maybe better to write:
```C++
constexpr Pager::Limit MAX_PAGER_LIMIT = iroha::model::Pager::MAX_PAGER_LIMIT;
```

Reply:
Can we assign the value in the old model file? I expect that this assertion will be removed when `makeOldModel()` and including old model file header are removed.